README.md

```diff
@@ -139,7 +210,8 @@ For collaborations please contact [Christos Kotselidis](https://www.kotselidis.n
 
 ## 10. TornadoVM Team
 
-This work was originated by James Clarkson under the joint supervision of [Mikel Luján](https://www.linkedin.com/in/mikellujan/) and [Christos Kotselidis](https://www.kotselidis.net).
+This work was originated by James Clarkson under the joint supervision
+of [Mikel Luján](https://www.linkedin.com/in/mikellujan/) and [Christos Kotselidis](https://www.kotselidis.net).
```

Comment:
we can remove that and put a link to the team at tornadovm.org @stratika

Reply:
I will redirect to https://www.tornadovm.org/about-us